### ISSUE TITLE
Empty States?

### ISSUE THREAD

**Issue by helensimonson** [hoist/Hoist#138]
When you create a new app and have no endpoints/schedules etc, maybe the boxes should have little messages in them saying something like (you currently have no endpoints, when you do this, they will show up here) etc.

**Comment by mejamiewilson** [comment]
Can you do a quick design for that and add the screenshot here? @helensimonson

**Comment by helensimonson** [comment]
![10 8 15 portal empty states-01](https://cloud.githubusercontent.com/assets/13734442/9177919/27261f06-3f49-11e5-8548-7c1331a2449f.png)
Maybe this is too wordy, but this general idea?

**Comment by mejamiewilson** [comment]
:+1:

**Comment by mejamiewilson** [comment]
Moving to backlog
